src/Analyzers/Analyzers/test/CSharpCodeFixVerifier`2+Test.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Testing;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Testing.Verifiers;
+
+namespace Microsoft.AspNetCore.Analyzers
+{
+    public static partial class CSharpCodeFixVerifier<TAnalyzer, TCodeFix>
+        where TAnalyzer : DiagnosticAnalyzer, new()
+        where TCodeFix : CodeFixProvider, new()
+    {
+        public class Test : CSharpCodeFixTest<TAnalyzer, TCodeFix, XUnitVerifier>
+        {
+            public Test()
+            {
+                ReferenceAssemblies = CodeAnalysis.Testing.ReferenceAssemblies.NetCore.NetCoreApp50;
+
+                foreach (var assembly in Directory.EnumerateFiles(AppContext.BaseDirectory, "*.dll"))
+                {
+                    TestState.AdditionalReferences.Add(CreateReferenceFromFile(assembly));
+                }
+
+                SolutionTransforms.Add((solution, projectId) =>
+                {
+                    var compilationOptions = solution.GetProject(projectId).CompilationOptions;
+                    compilationOptions = compilationOptions.WithSpecificDiagnosticOptions(
+                        compilationOptions.SpecificDiagnosticOptions.SetItems(CSharpVerifierHelper.NullableWarnings));
+                    solution = solution.WithProjectCompilationOptions(projectId, compilationOptions);
```

Comment:
Can you add a comment explaining what this does and why? Is this turning on `<Nullable>enable</Nullable>` for the project?

Reply:
I'll add a comment. The analyzer test framework verifies by default that the input sources do not contain compilation _errors_ (essentially warning level 0, expect for the analyzer(s) under test). Nullable reference types warnings don't show up in this case, which makes it very easy to accidentally write an incorrect test involving the feature.

This section of code elevates all warnings from this feature to errors, so they are validated by default test scenarios.